src/macaron/slsa_analyzer/provenance/intoto/v1/__init__.py

```diff
@@ -1,9 +1,37 @@
-# Copyright (c) 2023 - 2023, Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2023 - 2024, Oracle and/or its affiliates. All rights reserved.
 # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/.
 
 """This module handles in-toto version 1 attestations."""
 
-from typing import TypedDict
+from __future__ import annotations
+
+from collections.abc import Callable
+from typing import TypedDict, TypeGuard
+
+from macaron.slsa_analyzer.provenance.intoto.errors import ValidateInTotoPayloadError
+from macaron.util import JsonType
+
+VALID_ALGORITHMS = [
```

Comment:
Since this is only used in the `is_valid_digest_set` function, and the link to the documentation mentioning these algorithms is also there, do you think it is better to move this list into that function, for closer proximity to the documentation link? Either that or we may need to include the documentation link here because just by looking at this list, I feel it is a bit hard to tell where it came from and what it is for.